search/result.cpp

```diff
@@ -313,4 +317,15 @@ void AddressInfo::Clear()
   m_types.clear();
 }
 
+string DebugPrint(Result const & r)
```

Comment:
The main purpose of DebugPrint() is debug print, not serialization for statistics.